pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -157,23 +157,25 @@ func (gb *GraphBuilder) controllerFor(resource schema.GroupVersionResource, kind
 	return monitor, nil
 }
 
-func (gb *GraphBuilder) monitorsForResources(resources map[schema.GroupVersionResource]struct{}) error {
+func (gb *GraphBuilder) monitorsForResources(resources map[schema.GroupVersionResource]struct{}) {
 	for resource := range resources {
 		if _, ok := ignoredResources[resource]; ok {
 			glog.V(5).Infof("ignore resource %#v", resource)
 			continue
 		}
 		kind, err := gb.restMapper.KindFor(resource)
 		if err != nil {
-			return err
+			nonCoreMsg := fmt.Sprintf(nonCoreMessage, resource)
+			utilruntime.HandleError(fmt.Errorf("%v. %s", err, nonCoreMsg))
```

Comment:
If there are multiple restmapper failures and only one of them print the nonCoreMsg, readers may miss it. I think it's worthwhile to print nonCoreMsg multiple times, given that this function is only called at initialization time, so it won't spam the log much.